docs/notebooks/openai_gym_lunar_lander.pct.py

```diff
@@ -0,0 +1,253 @@
+# ---
```

Comment:
purely for consistency with other notebooks, lets remove this preamble

Reply:
ok

Reply:
Although I don't understad how the other notebooks don't have it. Jupytext seems to insert it by default when converting from ipynb. Am I the only one to create the notebooks as ipynb and then convert?